pythran/pythonic/numpy/expand_dims.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef PYTHONIC_NUMPY_EXPAND_DIMS_HPP
+#define PYTHONIC_NUMPY_EXPAND_DIMS_HPP
+
+#include "pythonic/include/numpy/expand_dims.hpp"
+#include "pythonic/numpy/ndarray/reshape.hpp"
+#include "pythonic/utils/functor.hpp"
+#include "pythonic/include/utils/array_helper.hpp"
+#include "pythonic/types/ndarray.hpp"
+
+PYTHONIC_NS_BEGIN
+
+namespace numpy
+{
+  template <typename T>
+  types::ndarray<typename T::dtype, types::array<long, T::value + 1>>
+  expand_dims(T const &input, int axis)
+  {
+    const int N = std::tuple_size<decltype(input._shape)>::value;
```

Comment:
could be `const long N = T::value`

Reply:
ah ah :) Thanks! There had to be a simpler way. You can see I'm still not familiar with the classes used in pythran!